### ISSUE TITLE
Cli - Refactoring with Argparse

### ISSUE THREAD

**Issue by BugliL** [Schrodinger-Hat/ImageGoNord#73]
I think the client is not so much pythonic, can I try to provide a version with argparse?

**Comment by Wabri** [comment]
Of course, I was already thinking about it, but other projects got in my way 😅

**Comment by Wabri** [comment]
@BugliL sorry for the shitty code 😬

**Comment by BugliL** [comment]
It Works so It's good, it just need a little bit  of refactoring to implement more options clearly

**Comment by Wabri** [comment]
@BugliL can this issue be closed?

**Comment by Artmorse** [comment]
Hi,

Before create a PR for the issue #64, i would like to close this one, I will propose you a PR very soon!

**Comment by Wabri** [comment]
Hi @Artmorse will you plan to use argpars in your pr?

**Comment by Wabri** [comment]
@BugliL have you done anything on this issue yet?

**Comment by Artmorse** [comment]
Yes I want to close this issue before starting the #64 one.

**Comment by BugliL** [comment]
> @BugliL have you done anything on this issue yet?

I have a new version on my repository that should be updated with last commits. It is quite ready.

**Comment by Artmorse** [comment]
If you need some help, don't hesistate!
